problem-16.c

```diff
@@ -0,0 +1,82 @@
+#include <stdio.h>
+#include <stdbool.h>
+// Basic calculator program that performs addition, subtraction, multiplication and division
+
+int getSum(int firstNum, int secondNum);
+int getDifference(int firstNum, int secondNum);
+int getProduct(int firstNum, int secondNum);
+int getQuotient(int firstNum, int secondNum);

```

Comment:
It's good to see that you have make this decision of putting functions below the code and handling the errors like this 👍 

I think it will be better if these function names matches with the menu that we are showing to users like `getSum` should be `getAddition`.